### PR TITLE
feat: initial fork release

### PR DESCRIPTION
**Pull request by skeggse**
- remove cli and local functionality
- remove typescript
- use standard Mixmax shared modules setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,122 +30,56 @@ npm install semantic-commitlint --save-dev
...
 
-## Config
+Simply run `semantic-release` as normal as part of your CI build process
 
-### Skip commits
+## Skip commits
 
-If there are unreleased commits that shouldn't fail a build, then add them to your `package.json` inside the `semanticCommitlint` config.
+If there are unreleased commits that shouldn't fail a build, then add them to the
+`SEMANTIC_COMMITLINT_SKIP` environment variable as comma-separated GIT SHAs. This should be
```
Comment (shils) [inline README.md]: Where would this environment variable be set?

Comment (shils) [inline README.md:54]: Is this something a developer would manually set in the `package.json`, or handled by CI somehow?

Comment (skeggse) [reply README.md:54]: This is something the developer would set temporarily in the `package.json`, or if they're using Travis, in the environment variables section. It's really only intended to help developers get this set up initially - the expectation is that they implement linting at the appropriate locations to ensure this never happens after the first release with this plugin.

**Comment by skeggse** [comment]
Merging for forward progress; happy to refactor/add tests etc one this is rolled out.

**Pull request merged**